application/src/main/java/run/halo/app/security/preauth/SystemSetupEndpoint.java

```diff
@@ -289,7 +289,7 @@ Flux<Unstructured> loadPresetExtensions(String username) {
                     var processedContent =
                         PROPERTY_PLACEHOLDER_HELPER.replacePlaceholders(rawContent, properties);
                     // load yaml to unstructured
-                    var stringResource = new InMemoryResource(processedContent);
+                    var stringResource = new InMemoryResource(processedContent.getBytes(StandardCharsets.UTF_8));
```

Comment:
Fix code style issue.

```suggestion
                    var stringResource = 
                        new InMemoryResource(processedContent.getBytes(StandardCharsets.UTF_8));
```